code/datums/dog_fashion.dm

```diff
@@ -157,7 +157,7 @@
 	..()
 	desc = "You must respect Elder [M.real_name]."
 
-/datum/dog_fashion/head/hop
+/datum/dog_fashion/head/xo //Nsv13 - HoP to XO
```

Comment:
Honestly it might be better to make children of basically all the stuff so its easier to manage as a separate file for conflict reasons. It is more annoying for mappers but we can work towards a easier system to manage that later on.

Reply:
now you tell me